js/index.js

```diff
@@ -2,38 +2,55 @@
 
 let weatherData;
 
-$(document).ready(() => {
+const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+const monthNames = ["January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"
+];
+
+const apiKey = "7cad625ece0a657d34524777c3f14cfa";
+const urlBase = "https://api.openweathermap.org/data/2.5";
+const urlWeather = `${urlBase}/weather`;
+const urlForecast = `${urlBase}/forecast`;
+
+
+
+$(document).ready(async () => {
     // TODO: Get city name from user/config
     let citi = "Warsaw";
 
+    await renderForecastData();
     document.getElementById("refresh-weather").onclick = () => (displayWeather(citi));
-    displayWeather(citi);
+    await displayWeather(citi);
```

Comment:
Nie widzę powodu, żeby użyć `await` oprócz ładowania DOM. Jeśli to jedyny powód to myślę, że to można poprawić projekt renderowania.